config/default.yml

```diff
@@ -532,6 +532,24 @@ Layout/SpaceInsideStringInterpolation:
     - space
     - no_space
 
+Layout/ClassStructure:
+  Description: 'Enforces a configured order of definitions within a class body.'
+  StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#consistent-classes'
```

Comment:
Omit except the anchor. `StyleGuide` attribute requires an anchor only, does not require a full URL.
For example: https://github.com/bbatsov/rubocop/blob/f9de7c34c71bb5dfa590a3c85ef84f382623a5de/config/enabled.yml#L7

Reply:
Awesome! Thanks, @pocke! I didn't know about and missed checking more examples. Fixed.